src/NServiceBus.Core/Settings/SettingsHolder.cs

```diff
@@ -130,6 +143,26 @@ public T GetOrDefault<T>(string key)
         }
 
         /// <summary>
+        /// Gets the default value for the setting or <code>default(T).</code>.
```

Comment:
How is it not obvious? it says 'Gets the default value for the setting' or 'default(T)'? how would you word it then?